packages/css/src/components/form/form-group.scss

```diff
@@ -37,10 +37,11 @@
   }
 
   .control-label {
-    display: inline-block;
+    display: inline;
     font-weight: bold;
     margin-bottom: $u1;
     max-width: 100%;
+    padding: 0;
```

Comment:
Welke padding zet je hiermee op `0`?

Reply:
Er werd een `padding-top` gezet in `packages\dso-toolkit\libs\bootstrap\_forms.scss`, maar daar wil je hem eigenlijk weghebben, ipv tegen te stijlen. Mee eens?

Reply:
Mee eens!